ci/script.sh

```diff
@@ -4,6 +4,10 @@ echo "inside $0"
 
 source activate pandas
 
+if [ x"$DOC_BUILD" != x"" ]; then
+    exit 0
```

Comment:
this one is on purpose and not for testing, so that the doc build does not run the tests (only the docs)